linux_os/guide/services/ntp/chronyd_no_chronyc_network/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Disable network management of chrony daemon'
+
+description: |-
+    The <tt>cmdport</tt> option in <tt>/etc/chrony.conf</tt> can be set to
+    <tt>0</tt> to stop chrony daemon from listening on the UDP port 323
+    for management connections made by chronyc.
+
+rationale: |-
+    Not exposing the management interface of the chrony daemon on
+    the network diminishes the attack space.
+
+severity: unknown
+
+identifiers:
+    cce@rhel8: 82840-0
```

Comment:
Should there be RHEL 7 or RHEL 6 CCEs?

Reply:
Possibly. I'm working based on the RHEL 8 kickstart by @stevegrubb, so not really focusing on older versions.
@yuumasato, do we want to add these things to older versions as well as part of the RHEL 8 review / work?

Reply:
@adelton If it is cheap, yes, we can add them, but I don't consider it essential for the PR.
What I consider important is the content itself, the rule, check and fix.

I'd like to avoid expanding the scope too much, and having to check configuration settings for three distributions.